sdk/tests/tutorials/ibor/test_valuation.py

```diff
@@ -102,22 +105,27 @@ def test_portfolio_aggregation(self):
                     default_scope=TestDataUtilities.tutorials_scope)
             )
         )
+        upsert_recipe_request = models.UpsertRecipeRequest(demo_recipe)
+        self.recipes_api.upsert_configuration_recipe(upsert_recipe_request)
 
-        aggregation_request = models.AggregationRequest(
-            inline_recipe=inline_recipe,
+        valuation_request = models.ValuationRequest(
+            recipe_id=models.ResourceId(scope=recipe_scope,code=recipe_code),
             metrics=[
                 models.AggregateSpec("Instrument/default/Name", "Value"),
                 models.AggregateSpec("Holding/default/PV", "Proportion"),
                 models.AggregateSpec("Holding/default/PV", "Sum")
             ],
             group_by=["Instrument/default/Name"],
-            effective_at=effective_date
+            portfolio_entity_ids = [
+                models.PortfolioEntityId(scope = TestDataUtilities.tutorials_scope, code = portfolio_code)
+            ],
+            valuation_schedule=models.ValuationSchedule(effective_at=effective_date)
         )
 
         #   do the aggregation
-        aggregation = self.aggregation_api.get_aggregation(scope=TestDataUtilities.tutorials_scope,
+        aggregation = self.aggregation_api.get_valuation(scope=TestDataUtilities.tutorials_scope,
```

Comment:
need to remove scope and code arguments here

Reply:
yes, done